package.json

```diff
@@ -57,6 +58,7 @@
     "eslint-plugin-ember": "^10.2.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
+    "husky": "^6.0.0",
```

Comment:
We won't be using `pre-commit` anymore, right? Can we have it uninstalled? Also the script on line 76-77-78 can be removed 🙂

Reply:
I thought of removing it after this PR getting accepted :sweat_smile:

Reply:
Will remove it now